### PR TITLE
fix(getstate): fix getState() on VTK objects

### PR DESCRIPTION
**Pull request by tomsuchel**
Commit ac8b26ce1f5c36f7c577f2363820416bd625a72b (merged in master, tag v24.0.0) made many model variables protected by suffixing them with _, but some variables have been forgotten. 
This commit fix this by renaming the remaining variables, mainly in the Widgets3D directory.

### PR and Code Checklist

- [x] [semantic-release](https://github.com/semantic-release/semantic-release) commit messages
- [x] Run `npm run reformat` to have correctly formatted code

### Testing

- [ ] This change adds or fixes unit tests 
- [x] Tested environment:
  - **vtk.js**: 24.0.0
  - **OS**: Ubuntu 20.04 LTS
  - **Browser**: Chromium 98.0.4697.0

### CODE REVIEW

**Comment by tomsuchel** [comment]
@finetjul

File: Sources/Widgets/Widgets3D/ShapeWidget/test/testShapeWidget.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
+import test from 'tape-catch';
```
Comment (finetjul) [inline Sources/Widgets/Widgets3D/ShapeWidget/test/testShapeWidget.js:1]: does not belong to this commit, does it ?

Comment (tomsuchel) [reply Sources/Widgets/Widgets3D/ShapeWidget/test/testShapeWidget.js]: No sorry, I've just removed it

**Comment by finetjul** [comment]
LGTM
